django/core/management/__init__.py

```diff
@@ -91,9 +91,9 @@ def call_command(command_name, *args, **options):
     testing.
 
     Some examples:
-        call_command('migrate')
-        call_command('shell', plain=True)
-        call_command('sqlmigrate', 'myapp')
+        call_command("migrate")
+        call_command("shell", plain=True)
+        call_command("sqlmigrate", "myapp")
```

Comment:
We have dozens or single quotes in docs and docstrings. I don't think it's worth changing.

Reply:
It's consistent with blacken-docs though? 🤷🏻‍♂️